db/migrate/20180621094208_add_correspondence_type_columns_to_report_types.rb

```diff
@@ -4,8 +4,6 @@ class AddCorrespondenceTypeColumnsToReportTypes < ActiveRecord::Migration[5.0]
   def up
     add_column :report_types, :foi, :boolean, default: false
     add_column :report_types, :sar, :boolean, default: false
-    ReportType.reset_column_information
```

Comment:
Bit worried about removing the seeder bit in this migration

Reply:
Adding a column to a table, and updating the seeds to populate that column cause problems when running rake db:reseed.  The way to do it is to maintain a strict separation of schema migrations and data migrations, so I've moved them all into data migrations.

Reseeding will run all the schema migrations, and then all the data migrations which will ensure everything is populated.  Ditto on production systems, we just run all the outstanding schema migrations first, followed by all the outstanding data migrations.